doc/fig-documentation/docs/guides/10-settings-binding-verification.md

```
@@ -0,0 +1,296 @@
+            Settings,
+            s => s.Location = $"Test-{Guid.NewGuid():N}",
+            s => s.Location);
+    }
+}
+```
+
+## Understanding Failures
+
+When verification fails, `FigSettingsBindingVerificationException` is thrown with a message that identifies the problem and hints at the fix:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)